content/docs/04-clusters/01-public-cloud/06-tke.md

```diff
@@ -160,21 +167,29 @@ A Tencent Cloud account needs to be created for the Palette Console from the Ten
     | **Secret Key**| The secret key of the Tencent cloud account.|
     
 
-3. Validate and confirm the given information to complete the wizard. 
+3. Click the **Validate** button to validate credentials.
+
+
+4. Click **Confirm** button to complete the cloud account create wizard. 
 
 
 **Note**: The Cloud Account can be created during the first step of cluster creation, when you fill in the basic information by clicking the **+** next to **Cloud Account**. 
 
-# Deploying a Tencent Cluster
+# Deploy a Tencent Cluster
 
 The following steps need to be performed to provision a new TKS cluster:
-1. Provide the basic cluster information like Name, Description, and Tags. Tags on a cluster are propagated to the VMs deployed on the cloud/data center environments.
 
+1. Provide the basic cluster information such as:
+   * **Name**, **Description**, and **Tags**. Tags on a cluster are propagated to the VMs deployed on the cloud/data center environments. 
+   * Select the desired [Tencent cloud account](/clusters/public-cloud/tke#createatencentcloudaccount). The Tencent credentials need to be preconfigured in the Project/Tenant Admin settings.
```

Comment:
```suggestion
   * Select the desired [Tencent cloud account](/clusters/public-cloud/tke#createatencentcloudaccount). 
```
The later can be removed because it's a prerequisite stated earlier.